src/pubsub/mod_pubsub_db_rdbms.erl

```diff
@@ -266,15 +266,17 @@ set_node(#pubsub_node{nodeid = {Key, Name}, id = undefined, type = Type,
                       owners = Owners, options = Opts, parents = Parents}) ->
     OwnersJid = [jid:to_binary(Owner) || Owner <- Owners],
     {ok, Nidx} = mod_pubsub_db_rdbms_sql:insert_pubsub_node(encode_key(Key), Name, Type,
-                                                     jsx:encode(OwnersJid),
-                                                     jsx:encode(Opts)),
+                                                     jiffy:encode({OwnersJid}),
```

Comment:
Why the surrounding `{}`. This produces different output the before. 
Hint: here, and in other such places we want to produce a list of strings.